homeworks/HW1/src/Courses.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import {Button, Form, FormGroup, Input, Col, Table, Row} from 'reactstrap';
+import {Link} from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import './App.css'
+
+export class Courses extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
You can use just a state without constructor if for you don't need to set an initial state which is not depended on props or other.
```
class Ddd extends React.Component {
   state = {
     courses: [],
   }
   ...
```